### ISSUE TITLE
EPUB Accessibility Metadata: should be version indipendent?

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/publ-a11y#63]
Right now all the metadata examples in EPUB Accessibility Metadata are for EPUB 3, we know that this metadata can also be included in EPUB 2.

Maybe put a note about this? Or include examples for EPUB 2 as well?
